TheWarWithin/DeathKnightFrost.lua

```diff
@@ -1980,6 +1992,14 @@ spec:RegisterSetting( "bos_rp", 60, {
     width = "full"
 } )
 
+spec:RegisterStateExpr("breath_ticks_left", function()
```

Comment:
Spacing.  Is this duplicated?  Only need to create it once.

Reply:
It sure is. I needed to de-declare it higher up and forgot to delete this one.